### DISCUSSION TITLE
Create "similar" custom exercises from built-in ones

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#123] [Ideas]
It would copy the type, the muscles and the image from a built-in exercise. Could be useful for custom exercises that are slight variations of built-in exercises, like "rope cable pushdown" or something along those lines.

**Comment by yrinkel** [comment]
Yes good idea. And spot-on example with the rope/V-var push down variations, as currently using tags to differentiate them but running into mix-ups of graph and history. 

“Similar” custom exercises would solve that problem. 

Same for kneeling crunches, and seated variations of exercises.

**Comment by gregwebs** [comment]
If I create a new exercise, I have to put in how the various muscle groups are used.
I also don't get a useful image.
What I do now is select the existing exercise and then in comments state what the actual exercise is.

I would like to be able to create an exercise with the data including muscle groups and image filled out from an existing exercise.
Alternatively there could be a first class concept of an exercise variation where an exercise inherits everything from an existing exercise but could override as needed.

**Comment by John-Gee** [comment]
I think the word you are looking for is 'Clone'.

**Comment by astashov** [comment]
This is done
